### PR TITLE
OKTA-700802 Update login.gov links

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Update login.gov IdP Authorization parameter links. Acrolinx fixes
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-700802](https://oktainc.atlassian.net/browse/OKTA-700802)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-logingov-idp/main/index.md](https://github.com/okta/okta-developer-docs/blob/290ab8d9ed6ca3ac8be20ee4714962200ce9e2ce/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-logingov-idp%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f324860c-b7d0-4141-8bd1-839272ef9a3f) | 67  | 63  | 68  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.38b7a20b-b28c-4558-a496-522cb2a88a04?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/add-logingov-idp/main/index.md [hunk 1]
```diff
@@ -23,13 +23,13 @@ Configure Login.gov as an external Identity Provider so that your users can quic
...
+Okta manages the connection to the Identity Provider (IdP) for your application. The connection sits between your application and the IdP that authenticates your users. The industry-standard term for this is Inbound Federation. Okta uses the OpenID Connect (OIDC) protocol with private key JWT for inbound federation with Login.gov. When a user signs in, you can link the user's Identity Provider account to an existing Okta user profile or choose to create a user profile using Just-In-Time (JIT) provisioning.
 
-> **Note:** We also support additional services such as directories and credential providers. See the [Okta Integration Network Catalog](https://www.okta.com/okta-integration-network/) to browse all integrations by use case.
+> **Note:** Okta also support other services, such as directories and credential providers. See the [Okta Integration Network Catalog](https://www.okta.com/okta-integration-network/) to browse all integrations by use case.
```
Comment (ethanhsiung-okta) [inline packages/@okta/vuepress-site/docs/guides/add-logingov-idp/main/index.md:28]: Should this be `Okta also supports other services` ?

**Review by ethanhsiung-okta — APPROVED** [review]
1 comment, otherwise looks good to me

File: packages/@okta/vuepress-site/docs/guides/add-logingov-idp/main/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/add-logingov-idp/main/index.md:28]:
```suggestion
> **Note:** Okta also supports other services, such as directories and credential providers. See the [Okta Integration Network Catalog](https://www.okta.com/okta-integration-network/) to browse all integrations by use case.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/add-logingov-idp/main/index.md](https://github.com/okta/okta-developer-docs/blob/ed9a20adf1af4acffbce69bed7f99e01f004df9c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fadd-logingov-idp%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/265b0847-1fb4-4383-a8e4-a900a4e6f491) | 67  | 65  | 68  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.0934dcbd-93e9-44da-b007-fd81e5fefaea?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**